pkg/controllers/dashboard/fleetcharts/controller.go

```diff
@@ -107,6 +109,14 @@ func (h *handler) onSetting(key string, setting *v3.Setting) (*v3.Setting, error
 		gitjobChartValues["noProxy"] = envVal
 	}
 
+	// add priority class value
+	if priorityClassName, err := h.chartsConfig.GetPriorityClassName(); err != nil {
+		logrus.Warnf("Failed to get rancher priorityClassName: %v", err)
+	} else {
+		fleetChartValues["priorityClassName"] = priorityClassName
```

Comment:
nit: could use a const like you do here https://github.com/rancher/rancher/pull/39071/files#diff-850ef33e7865013ff6b5d6ef1ea797d9651658fb7a42ec5a5b8dbbdf15249937R13